Conventional.Samples/1 - Domain/Attendee.cs

```diff
@@ -19,8 +19,8 @@ public static Attendee Create(Guid id)
             return new Attendee(id);
         }
 
-        public Guid Id { get; }
+        public Guid Id { get; private set; }
```

Comment:
The `PropertiesMustHavePrivateSetters` convention requires this - I'm not actually sure if that's "right", given backing fields/readonly props, but that's probably not a discussion for here!